tests_v2/unit/app/routes/geostore/test_geostore.py

```diff
@@ -178,7 +184,95 @@
 
 
 @pytest.mark.asyncio
-async def test_get_admin_geostore(async_client: AsyncClient, monkeypatch: MonkeyPatch):
+async def test_get_admin_country_geostore_gfw_branch(
+    async_client: AsyncClient, monkeypatch: MonkeyPatch
+):
+    # CursorResult = MagicMock(cursor)
```

Comment:
comment to delete? or is it for future debugging?

Reply:
Yep, will get rid of it.